cratedb_sqlparse_py/cratedb_sqlparse/parser.py

```diff
@@ -169,9 +170,10 @@ def find_suitable_error(statement, errors):
         # we can get the full query in the error handler but not in the context.
         error_query = error.query
         if error_query.endswith(";"):
-            error_query = error_query[: len(error_query) - 1]
+            error_query = error_query[:len(error_query) - 1]
 
-        if error_query.lstrip().rstrip() == statement.query:
+        error_query = error_query.lstrip().rstrip()
+        if error_query == statement.query or statement.query in error_query:
```

Comment:
Is it effectively the same?
```suggestion
        if statement.query in error_query:
```

Reply:
Yes! Thanks